pkg/cluster/cluster.go

```diff
@@ -270,10 +286,11 @@ func New(ctx context.Context, log *logrus.Entry, _env env.Interface, db database
 		graph:   graph.NewManager(_env, log, aead, storage),
 		rpBlob:  rpBlob,
 
-		installViaHive:                    installViaHive,
-		adoptViaHive:                      adoptByHive,
-		hiveClusterManager:                hiveClusterManager,
-		now:                               func() time.Time { return time.Now() },
-		openShiftClusterDocumentVersioner: new(openShiftClusterDocumentVersionerService),
+		installViaHive:                         installViaHive,
+		adoptViaHive:                           adoptByHive,
+		hiveClusterManager:                     hiveClusterManager,
+		now:                                    func() time.Time { return time.Now() },
```

Comment:
Do we need Now on line 128 and here?

Reply:
Yes I think so, this part hasn't been touched in this PR. And now is being used here:- https://github.com/Azure/ARO-RP/blob/master/pkg/cluster/install.go#L415